admin_ui/app/dashboard/DashboardPage.js

```diff
@@ -1,81 +1,128 @@
-
-// TODO: Fetch actual data and replace this
-const dummyOrgs = [
-  {
-    id: 1,
-    name: 'Our Tech Blog',
-    users: 10,
-    activeSessions: 0,
-    roles: 2,
-  },
-  {
-    id: 2,
-    name: 'Zoho CRM',
-    users: 30,
-    activeSessions: 1,
-    roles: 7,
-  },
-];
-
-const DashboardOrgCard = (props) => {
-  return (
-    <div className="border border-grey rounded p-4 flex-1 mb-4 sm:mr-4 relative">
-      <h2 className="mb-4 font-bold text-2xl">
-        <Link to={`/organizations/${props.id}/edit`}>{props.name}</Link>
-      </h2>
-      <IconOrganisation className="absolute top-0 right-0 mt-4 mr-4" height={28} />
-      <table className="w-full">
-        <tbody>
-          <tr>
-            <td>Total users:</td>
-            <td className="text-right">
-              <Link to="/users">{props.users}</Link>
-            </td>
-          </tr>
-          <tr>
-            <td>Active sessions:</td>
-            <td className="text-right">{props.activeSessions}</td>
-          </tr>
-          <tr>
-            <td>Org-specific roles:</td>
-            <td className="text-right">
-              <Link to="/roles">{props.roles}</Link>
-            </td>
-          </tr>
-        </tbody>
-      </table>
-    </div>
-  );
-};
-DashboardOrgCard.propTypes = {
-  id: PropTypes.number.isRequired,
-  name: PropTypes.string.isRequired,
-  users: PropTypes.number,
-  activeSessions: PropTypes.number,
-  roles: PropTypes.number,
-};
+import IconUser from '../../icons/IconUser';
+import ButtonLink from '../../components/ButtonLink';
+import IconPermission from '../../icons/IconPermission';
+import IconRole from '../../icons/IconRole';
+import { useSelector, useDispatch } from 'react-redux';
+import { listOrgs } from '../org/orgStore';
+import { listPermissions } from '../permission/permissionStore';
+import { listRoles } from '../role/roleStore';
+import Spinner from '../../components/Spinner';
 
 const DashboardPage = () => {
   useDocumentTitle('Dashboard');
+  const dispatch = useDispatch();
+
+  // TODO:
+  //  Loading all data for 3 models (4 after users are ready) is a bit redundant.
+  //  Consider implementing a .getCounts() API call
+  const orgCountLoading = useSelector((state) => state.org.list.isLoading);
+  const orgCount = useSelector((state) => state.org.list.totalCount);
+  const roleCountLoading = useSelector((state) => state.role.list.isLoading);
+  const roleCount = useSelector((state) => state.role.list.totalCount);
+  const permissionCountLoading = useSelector((state) => state.permission.list.isLoading);
+  const permissionCount = useSelector((state) => state.permission.list.totalCount);
+  // TODO: Populate the two varibles below with proper selectors
+  //  once branch user-adminui-management is merged
+  const userCountLoading = false;
+  // Temporary: Change to 1 to view the "normal" dashboard view.
+  // Keep as 0 to view the onboarding UI
+  const userCount = 0;
```

Comment:
This needs to be removed / replaced with looking up the value in the store once the Users branch is merged.